src/SampleBaseApp.ts

```diff
@@ -37,26 +39,34 @@ export class SampleBaseApp {
     // initialize OIDC
     await SampleBaseApp.initializeOidc();
 
+    // use new state manager that allows dynamic additions from extensions and snippets
+    if (!this._appStateManager) {
+      this._appStateManager = new StateManager({
+        frameworkState: FrameworkReducer,
+      });
+    }
+
     // contains various initialization promises which need
     // to be fulfilled before the app is ready
     const initPromises = new Array<Promise<any>>();
 
     // initialize RPC communication
     initPromises.push(SampleBaseApp.initializeRpc());
 
-    // initialize UiCore
-    // initPromises.push(UiCore.initialize(IModelApp.i18n));
-
-    // initialize UiComponents
-    initPromises.push(UiComponents.initialize(IModelApp.i18n));
+    // initialize UiFramework
+    initPromises.push (UiFramework.initialize(undefined));
 
     // initialize Presentation
     initPromises.push(Presentation.initialize({
       activeLocale: IModelApp.i18n.languageList()[0],
     }));
 
+    // initialize Markup
+    initPromises.push(MarkupApp.initialize());
```

Comment:
Markup is a dependency introduced by UiFramework. I can try running without initializing it and let you know.

Reply:
I talked with Bill about Markup. This is a legacy from 1.x, so that apps (specifically) could have standard definitions for the Ui associated with tools delivered with the imodel.js frontend. We need to initialize markup for the default tool widgets. I'm certain the desktop starter does something similar (either initialize or start).